NEWS.md

```diff
@@ -1,3 +1,11 @@
+# ggplot2 2.1.0.9000
+
+## Bug fixes and minor improvements
```

Comment:
Sorry to keep nit picking, but I prefer to add the headings just before release.